organization-understanding.md

```diff
@@ -0,0 +1,76 @@
+# Memorandum of Understanding
+
+**This is a draft.**
+
+[NumFOCUS](http://numfocus.org/)
+will apply to Google Summer of Code (GSoC)
+as a umbrella organization.
+[All projects/organizations supported by NumFOCUS](http://numfocus.org/projects/)
+can participate at GSoC under NumFOCUS umbrella.
+Projects/organizations not supported by NumFOCUS
+can participate at GSoC under NumFOCUS umbrella
+if approved by NumFOCUS board.
+
+## NumFOCUS Administrator for GSoC
+
+The NumFOCUS board will indicate one person,
+named in this document as **organization administrator**,
+to be in charge of NumFOCUS application **for each edition** of GSoC.
+
+The **organization administrator** is responsible for
+
+-   create the initial application for NumFOCUS to apply for GSoC
+-   advertise the application to **all** projects/organization supported
+    by NumFOCUS
+-   manage NumFOCUS profile at GSoC
+-   request a minimum and a maximum number of slots
+    that can accomodate the slots requested by each project/organization
+    under NumFOCUS umbrella.
```

Comment:
> We also need a backup administrator.

Yes. I thought that one coordinator could be the backup administrator. @aterrel Do you prefer that NumFOCUS board name the backup administrator?